hypha/apply/users/views.py

```diff
@@ -241,7 +278,7 @@ def get(self, request, *args, **kwargs):
         if self.valid(user, kwargs.get('token')):
             user.backend = settings.CUSTOM_AUTH_BACKEND
             login(request, user)
-            return redirect('users:activate_password')
+            return redirect('users:account')
```

Comment:
Curious why this change is needed?

Reply:
This change was based on the workflow that we adopted here, were three things are set:

1. We require login to make an application
2. You can register before making an application (forced by 1.)
3. Local passwords are used, and set during registration (as opposed to using 2FA)

What that means is that upon activating your account, you've already set your password.  So you don't need to do that again.  You should just get redirected to your account page.

I can understand how this change is actually bad when and this change got missed when cherry picking our old internal commits, so it should definitely be removed (since the default behavior should remain unchanged).  But maybe the correct answer is to do some kind of switch depending on whether the user set the password, or it was set to a random one by the system?

What do you think?

Reply:
I think we can put this as conditional under the new setting introduced by you. So it retains the old behavior and updates only when login/register is required before submission. 

@frjo thoughts?

Reply:
There's also the path where registration is enabled, but passwords aren't part of it (and are either set later or they use 2FA).  So maybe conditional on both of those, otherwise the user gets sent to the activation page.

So both `WAGTAIL_PASSWORD_MANAGEMENT_ENABLED` and `ENABLE_REGISTRATION_WITHOUT_APPLICATION`

Reply:
@frankduncan That sounds good to me. Let's have that in.

Reply:
All set!  Though it turned out to be `WAGTAILUSERS_PASSWORD_ENABLED`, not `WAGTAIL_PASSWORD_MANAGEMENT_ENABLED` for the password fields to be available during registration